Adding SVI CRUD Tests for .NET SDK Testing (#29491)
Azure/azure-sdk-for-net: sdk/workloads/Azure.ResourceManager.Workloads/tests/SetupInstructions.md

# Setup Instructions

> This is a quick guide to running the tests in this repo. 

Reviewer: Nit:
```suggestion
This is a quick guide to running the tests of Svi. 
```
